### PR TITLE
Circumstellar zones

### PR DESCRIPTION
**Pull request by spadino**
Added metalrich, water worlds, earth like and ammonia worlds zones to JournalScan.

Included support for new zones in ScanGrid (which also received a brand new contextmenu) and SPanel.

### CODE REVIEW

File: EDDiscovery.sln.DotSettings [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (klightspeed) [inline EDDiscovery.sln.DotSettings]: Probably should .gitignore `*.sln.DotSettings`

File: EDDiscovery/EDDiscovery.csproj [hunk 2]
```diff
@@ -243,7 +243,6 @@
       <SubType>Component</SubType>
     </Compile>
     <Compile Include="EDDProfiles.cs" />
-    <Compile Include="Properties\ExtraVersion.cs" />
```
Comment (klightspeed) [inline EDDiscovery/EDDiscovery.csproj]: People really don't want build versioning, do they?

File: EDDiscovery/EDDiscovery.csproj [hunk 3]
```diff
@@ -535,11 +534,11 @@
     <Compile Include="UserControls\UserControlScan.Designer.cs">
       <DependentUpon>UserControlScan.cs</DependentUpon>
     </Compile>
-    <Compile Include="UserControls\UserControlSpanel.cs">
+    <Compile Include="UserControls\UserControlSPanel.cs">
```
Comment (klightspeed) [inline EDDiscovery/EDDiscovery.csproj]: This rename (without the corresponding file rename) is causing a travis build failure

File: EDDiscovery/EDDiscovery.csproj (hunk 2 above)
Comment (iainross) [reply EDDiscovery/EDDiscovery.csproj]: Just add in an empty file with the right name locally - it's included in the .gitignore so you won't commit the file itself but you'll stop the .csproj change being included in the commit.

**Comment by spadino** [comment]
Closing so to fix and revert some changes.

**Pull request closed without merging**